modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java

```diff
@@ -29,13 +30,15 @@
     public static final String IMAGE = "cassandra";
 
     public static final Integer CQL_PORT = 9042;
+    public static final String DEFAULT_LOCAL_DATACENTER = "datacenter1";
     private static final String CONTAINER_CONFIG_LOCATION = "/etc/cassandra";
     private static final String USERNAME = "cassandra";
     private static final String PASSWORD = "cassandra";
 
     private String configLocation;
     private String initScriptPath;
     private boolean enableJmxReporting;
+    private String localDatacenter;
```

Comment:
this variable seems to be write-only. It does not end up in container's configuration (e.g. environment variable). Let's either make it affect the container or remove.